src/docs/asciidoc/cdc-configuration.adoc

```diff
@@ -595,6 +595,87 @@ The Eventuate Tram CDC supports `eventuate-tram` and `eventuate-local`
 
 |===
 
+==== Configuring a cleaner
+
+Cdc can automatically remove old messages from message table and received_message table.
+Example configuration:
+
+----
+EVENTUATE_CDC_CLEANER_C1_PIPELINE: P4
+EVENTUATE_CDC_CLEANER_C1_MESSAGE_CLEANING_ENABLED: "true"
+EVENTUATE_CDC_CLEANER_C1_MESSAGES_MAX_AGE_IN_SECONDS: 1
+EVENTUATE_CDC_CLEANER_C1_INTERVAL_IN_SECONDS: 1
+...
+----
+
+If pipeline is specified, pipeline database configuration is used.
```

Comment:
Why would the user NOT want to specify the pipeline?

Reply:
I don't know. But probably we discussed this option when I was working on implementation.